client/src/css/structure.css

```diff
@@ -888,19 +888,35 @@ a[disabled] {
   background-color: #ff000083 !important;
 }
 
+.icon-expired {
+  color: #ff000083 !important;
+}
+
 .at-risk {
   background-color: #ffff6686 !important;
 }
 
+.icon-at-risk {
+  color: #ffff6686 !important;
+}
+
 .at-risk-of-expiring {
   background-color: #ff880191 !important;
 }
 
+.icon-at-risk-of-expiring {
+  color: #ff880191 !important;
+}
+
 .out-of-stock {
   /* text-decoration : line-through; */
   background-color: #cc00ff8f !important;
 }
 
+.icon-out-of-stock {
+  color: #cc00ff8f !important;
```

Comment:
I encourage you to just use the 6-digit version here.  You do not need or want any transparency in the legend!